compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/codegen/irCodegenUtils.kt

```diff
@@ -304,6 +253,8 @@ fun IrFunction.isInlineOnly() =
 
 fun IrFunction.isReifiable() = typeParameters.any { it.isReified }
 
+private fun IrFunction.isBridge() = origin == IrDeclarationOrigin.BRIDGE || origin == IrDeclarationOrigin.BRIDGE_SPECIAL
```

Comment:
This is a really minor nitpick, but maybe make this `internal` and use it in `FunctionCodegen.calculateMethodFlags`? And possibly also in `MethodSignatureMapper.mapOverriddenSpecialBuiltinIfNeeded`

Reply:
Good idea, done.